spec/cli/commands/download_spec.rb

```diff
@@ -1,7 +1,67 @@
 require 'spec_helper'
 require 'ronin/wordlists/cli/commands/download'
+require 'ronin/wordlists/root'
 require_relative 'man_page_example'
 
 describe Ronin::Wordlists::CLI::Commands::Download do
   include_examples "man_page"
+
+  let(:url)         { 'http://example.com/wordlist.txt' }
+  let(:yaml_result) { { 'download_wordlist' => { url: url } } }
```

Comment:
I try to avoid too many `{` and `}` on one line for readability. Maybe change it to `let(:...) do ... end`.